src/cloudant/database.py

```diff
@@ -214,51 +214,40 @@ def get_design_document(self, ddoc_id):
-        :class:`~cloudant.result.Result` which uses the ``skip`` and
-        ``limit`` query parameters internally to handle slicing and iteration
-        through the view result collection.  Therefore ``skip`` and ``limit``
-        cannot be used as arguments to get the view result when
-        ``raw_result=False``.  However, by setting ``raw_result=True``, the
-        result will be returned as the raw JSON response content for the view
-        requested.  Using this setting requires the developer to manage their
-        own slicing and iteration.  Therefore ``skip`` and ``limit`` are valid
-        arguments in this instance.
+        Retrieves the view result based on the design document and view name.
+        By default the result is returned as a
+        :class:`~cloudant.result.Result` object which provides a key
+        accessible, sliceable, and iterable interface to the result collection.
+        Depending on how you are accessing, slicing or iterating through your
+        result collection certain query parameters are not permitted.  See
+        :class:`~cloudant.result.Result` for additional details.
+
+        However, by setting ``raw_result=True``, the result will be returned as
+        the raw JSON response content for the view requested.  With this setting
+        there are no restrictions on the query parameters used but it also
+        means that the result collection key access, slicing, and iteration is
+        the responsibility of the developer.
 
-        For example to retrieve the default Result object based on a
-        design document view do:
+        For example:
 
         .. code-block:: python
 
-            db.get_view_result('_design/ddoc_id_001', 'view_001')
-
-        But to retrieve a customized Result object based on the
-        same design document view do something like:
+            # get Result based on a design document view
+            result = db.get_view_result('_design/ddoc_id_001', 'view_001')
 
-        .. code-block:: python
-
-            db.get_view_result('_design/ddoc_id_001', 'view_001',
+            # get a customized Result based on a design document view
+            result = db.get_view_result('_design/ddoc_id_001', 'view_001',
                 include_docs=True, reduce=False)
 
-        To retrieve the raw JSON response content based on a
-        design document view do:
-
-        .. code-block:: python
-
-            db.get_view_result('_design/ddoc_id_001', 'view_001',
+            # get raw response content based on a design document view
+            result = db.get_view_result('_design/ddoc_id_001', 'view_001',
                 raw_result=True)
 
-        But to retrieve the raw JSON response content based on a set of
-        parameters on the same design document view do something like:
-
-        .. code-block:: python
-
+            # get customized the raw response content for a design document view
```

Comment:
`get customized the raw response content` sounds weird. 
Maybe something like:
`get raw response with customized content based on a design doc view`